neqo-http3/tests/priority.rs

```diff
@@ -129,7 +129,7 @@ fn priority_update_dont_send_for_cancelled_stream() {
             Instant::now(),
             "GET",
             &("https", "something.com", "/"),
-            &[],
+            &[Header::new("priority", "u=5")],
```

Comment:
Ditto.

Reply:
This test doesn't check the headers as far as I can tell, but I put them here for consistency.